### PR TITLE
7654 update 404 error page

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-7654

### CODE REVIEW

File: source/404.html.erb [hunk 1]
```diff
@@ -3,6 +3,14 @@ title: Codio - 404 Not found
 layout: error
 ---
 
-<h2>
-  We are very sorry, but this is not the page you are looking for.
-</h2>
+<div class="error-container">
+	<div class="illustration">
```
Comment (AndreyNikitin) [inline source/404.html.erb]: too many tabs?

File: source/javascripts/partials/_header.es6 [hunk 2]
```diff
@@ -43,8 +48,13 @@ $(() => {
         $subHeader.hide();
         $header.height(65);
       } else {
-        $subHeader.show();
-        $header.height(105);
+        if ($(window).width() > 800) {
```
Comment (AndreyNikitin) [inline source/javascripts/partials/_header.es6]: same code used twice.

File: source/404.html.erb (hunk 1 above)
Comment (denpira) [inline source/404.html.erb]: Done

File: source/javascripts/partials/_header.es6 (hunk 2 above)
Comment (denpira) [inline source/javascripts/partials/_header.es6]: Done

**Comment by AndreyNikitin** [comment]
:+1:

File: source/stylesheets/pages/_error.scss [hunk 3]
```diff
@@ -1,10 +1,75 @@
+body.x404 {
+  .content {
+    margin: 0 auto 0 auto;
```
Comment (AndreyNikitin) [inline source/stylesheets/pages/_error.scss]: margin: 0 auto;

**Comment by AndreyNikitin** [comment]
:+1:

**Pull request merged**